### PR TITLE
Add details about new workflow to add files in .semgrepignore through App

### PR DESCRIPTION
**Pull request by adamkvitek**
# Thanks for improving Semgrep Docs 😀

### Please ensure

- [x] A subject matter expert (SME) reviews the content
- [x] A technical writer reviews the content or PR
- [x] This change has no security implications or else you have pinged the security team
- [ ] Redirects are added if the PR changes page URLs

### CODE REVIEW

File: docs/ignoring-files-folders-code.md [hunk 1]
```diff
@@ -66,47 +64,54 @@ Semgrep provides several methods to customize ignore behavior. Refer to the foll
 
 | Goal | Method |
 |:---- |:------ |
-| To scan all files within Semgrep's scope each time you run Semgrep (only files within `.git` will be ignored). | Create an empty `.semgrepignore` file in your repository root directory or in your project's working directory. An empty `.semgrepignore` will make Semgrep scan paths in `.gitignore`. |
+| To scan all files within Semgrep's scope each time you run Semgrep (only files within `.git` are ignored). | Create an empty `.semgrepignore` file in your repository root directory or in your project's working directory. If your `.semgrepignore` file is empty, Semgrep scans only paths in `.gitignore`. |
```
Comment (sabrinabrogren) [inline docs/ignoring-files-folders-code.md:67]: Is this change here correct? That if the `.semgrepignore` is blank, semgrep will only scan the paths in the `.gitignore`? That doesn't seem right.

File: src/components/procedure/_remove-rule.mdx [hunk 2]
```diff
@@ -6,13 +6,13 @@ To **remove a rule** from the Rule board:
...
 :::info
-- Individual rules within rulesets can only be disabled, not deleted. To disable an individual rule, click the <i class="fa-light fa-toggle-large-on"></i> toggle to disable the rule.
+- Individual rules within rulesets can only be disabled, not deleted. To disable an individual rule in **[Rule board](https://semgrep.dev/orgs/-/board)**, click the <i class="fa-light fa-toggle-large-on"></i> toggle to disable the rule.
 - When you remove a rule from the Rule Board, all associated findings on Findings page and Dashboard page are removed also.
 :::
 
 You can also remove a rule on the Findings page, to do so, follow these steps:
 
-1. Go to the [Findings](https://semgrep.dev/orgs/-/findings?tab=open) page.
+1. On the Semgrep App [Findings](https://semgrep.dev/orgs/-/findings?tab=open) page.
```
Comment (pabloest) [inline src/components/procedure/_remove-rule.mdx:15]: I think this needs a verb. Maybe “Start on the…”?

**Review by pabloest — APPROVED** [review]
Left one minor comment

**Review by sabrinabrogren — APPROVED** [review]

**Comment by sabrinabrogren** [comment]
Looks good to me overall, just left one comment on something that might need to be corrected

File: docs/ignoring-files-folders-code.md (hunk 1 above)
Comment (adamkvitek) [reply docs/ignoring-files-folders-code.md:67]: Thanks for a good catch @sabrinabrogren! The original is not right as well: `An empty `.semgrepignore` will make Semgrep scan paths in `.gitignore`.` Maybe it means, that when you do not include an empty `.semgreignore` file, Semgrep will use `.gitignore` instead to see what it has to ignore?

File: src/components/procedure/_remove-rule.mdx (hunk 2 above)
Comment (adamkvitek) [reply src/components/procedure/_remove-rule.mdx:15]:
```suggestion
1. Go to the Semgrep App [Findings](https://semgrep.dev/orgs/-/findings?tab=open) page.
```

File: docs/ignoring-files-folders-code.md (hunk 1 above)
Comment (sabrinabrogren) [reply docs/ignoring-files-folders-code.md:67]: That would make sense to me. Let me post this in the #team-app channel, I'm not sure about this.

Comment (adamkvitek) [reply docs/ignoring-files-folders-code.md:67]:
```suggestion
| To scan all files within Semgrep's scope each time you run Semgrep (only files within `.git` are ignored). | Create an empty `.semgrepignore` file in your repository root directory or in your project's working directory. An empty `.semgrepignore` will make Semgrep scan paths in `.gitignore`. |
```

File: docs/ignoring-files-folders-code.md [hunk 3]
```diff
@@ -29,27 +30,45 @@ Without user customization, Semgrep refers to the following to define ignored fi
...
+test/
+tests/
+*_test.go
+
+# Semgrep rules folder
+.semgrep
+
+# Semgrep-action log folder
+.semgrep_logs/
+
```
Comment (adamkvitek) [inline docs/ignoring-files-folders-code.md:57]:
```suggestion
DEFAULT_SEMGREPIGNORE_TEXT
```

File: docs/extensions.md [hunk 4]
```diff
@@ -25,7 +25,7 @@ The [pre-commit framework](https://pre-commit.com/) can run `semgrep` at commit-
 ```yaml
 repos:
 - repo: https://github.com/returntocorp/semgrep
-  rev: 'SEMGREP_VERSION_LATEST'
+  rev: 'v0.122.0'
```
Comment (adamkvitek) [inline docs/extensions.md:28]:
```suggestion
  rev: 'SEMGREP_VERSION_LATEST'
```

**Pull request merged**